q2_qemistree/_semantics.py

```diff
@@ -12,10 +12,61 @@
 import os
 
 
+def validate_mgf(iterable):
+    observed_ids = set([])
+
+    # ms1 and ms2 count the number of records for the MS1 and MS2 spectra
+    read_id, new_feature_id, ms1, ms2 = None, None, 0, 0
+
+    for line in iterable:
+        if line.startswith('FEATURE_ID='):
+            # get the feature identifier without the new line
+            new_feature_id = line.split('=')[1].strip()
+
+            if read_id is None:
+                read_id = new_feature_id
+            else:
+                if new_feature_id != read_id:
+                    if ms2 < 1:
+                        raise ValueError('Feature "%s" does not have at least '
```

Comment:
Can we replace this by "At least one feature (Feature ID = XYZ) does not have MS2 information" & make this a warning?